packages/tasks_repository/tasks_data.dart

```diff
@@ -0,0 +1,30 @@
+class Task {
+final String title;
+final String description;
+final bool isDone;
+
+Task({
+required this.title,
+required this.description,
+required this.isDone,
+});
+}
+
+List tasks = [
```

Comment:
One more thing, Avoid creating a public variables. Let's try to store all the variables inside a class. In this case you could create a class called TasksDevData and store the list of tasks there as a static const.